configure/create-immutable-and-secrets.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Missing from this description is when this script needs to be run.

I assume this should only be run when creating a new cluster.